### PR TITLE
new module ts_simple_percentile

### PR DESCRIPTION
**Pull request by hmmr**
RTS-545, RTS-1165 (MEDIAN); RTS-1173, RTS-1553 (PERCENTILE); RTS-547, RTS-1222 (MODE)

Depends on https://github.com/basho/riak_kv/pull/1624.

Tests for inverse distribution functions: PERCENTILE_DISC, PERCENTILE_CONT, MEDIAN, MODE.

### CODE REVIEW

**Comment by gordonguthrie** [comment]
Is this PR agains the right base? surely `riak_ts-develop`?

**Comment by hmmr** [comment]
Changed base.

File: bin/rtdev-setup-releases.sh [hunk 1]
```diff
@@ -39,20 +39,16 @@ cd $RT_DEST_DIR
...
 ## Some versions of git and/or OS require these fields
-HAS_LOCAL=$(git config --local 2>&1 | grep unknown)
-if [ -z "$HAS_LOCAL" ]; then
-    git config --local user.name "Riak Test"
-    git config --local user.email "dev@basho.com"
-    git config --local core.autocrlf input
-    git config --local core.safecrlf false
-    git config --local core.filemode true
-fi
+git config --local user.name "Riak Test"
```
Comment (hazen) [inline bin/rtdev-setup-releases.sh]: Can you please revert these changes to this script? It is supposed to work on different versions of git now

**Review by hazen — CHANGES_REQUESTED** [review]
Just need to switch back changes to a bash script

**Review by hazen — APPROVED** [review]

**Comment by macintux** [comment]
Would suggest including a test of a table where the column to be measured is defined as descending in the DDL.

**Comment by hmmr** [comment]
@macintux now testing on tables with keys declared DESC, in aa4b672.

**Comment by macintux** [comment]
+1

**Pull request merged**